src/components/dialogs/two-windings-transformer/create-rule-dialog.js

```diff
@@ -38,10 +39,9 @@ export const CreateRuleDialog = (props) => {
     });
 
     const isTapValuesValid = () => {
-        if (highTapValue && lowTapValue) {
-            return !(highTapValue - lowTapValue > 0);
-        }
-        return true;
+        return (
+            !highTapValue || !lowTapValue || highTapValue - lowTapValue === 0
```

Comment:
I doubt this is a correct expression for taps validity.
For example, having highTapValue equal to 0 would validate any lowTapValue.
Don't think this is what is required.

Reply:
I checked and for highTapValue and lowTapValue, they must be positive in ratio tap but can be negative in phase tap.
I made the changes